Makefile

```diff
@@ -135,6 +135,10 @@ install-core: all
 	done
 	printf "libexecdir = '%s'\nsysconfdir = '%s'\n" \
 	    "$(libexecdir)" "$(sysconfdir)" > $(DESTDIR)$(pythondir)/_stbt/vars.py
```

Comment:
Do we still need to record `libexecdir`? Yes we do, for `_stbt/control.py::stbt_control_listen`, which imports `$libexecdir/stbt/stbt-control`, and which is used by `stbt record --control-recorder`.

Reply:
The real fix there is to move (at least some of) the contents of `stbt-control` into a library under `_stbt`.  I couldn't be bothered to untangle it right now though.